src/operations/resize/mod.rs

```diff
@@ -40,7 +40,13 @@ impl OperationsTrait for Resize {
 
     fn execute_impl(&self, image: &mut Image) -> Result<(), ImageErrors> {
         let (src_width, src_height) = image.dimensions();
-        let (dst_width, dst_height) = self.new_dimensions;
+        let (mut dst_width, mut dst_height) = self.new_dimensions;
```

Comment:
Actually, this is not elegant, but it's simple and easy to implement

Reply:
Should this be improved?

Reply:
I think we should leave an error if the user enters 0 as width or height. We will not change the input values ​​as this may confuse the user

Reply:
ok, i'll replace it with a panic!